themes/digital.gov/layouts/events/single.html

```diff
@@ -146,6 +141,8 @@ <h1>{{- .Title | markdownify -}}</h1>
 
             <div class="dg-sidebar grid-col-12 tablet:grid-col-3">
               {{/* Authors */}}
+              {{- partial "core/primary_communities.html" . -}}
+
```

Comment:
This should come after the authors section. See live site — https://digital.gov/event/2022/05/17/how-to-build-your-digital-dream-team/